compiler/rustc_target/src/spec/aarch64_apple_ios.rs

```diff
@@ -2,8 +2,9 @@ use super::apple_sdk_base::{opts, Arch};
 use crate::spec::{FramePointer, Target, TargetOptions};
 
 pub fn target() -> Target {
+    let llvm_target = super::apple_base::ios_llvm_target("arm64");
     Target {
-        llvm_target: "arm64-apple-ios".to_string(),
+        llvm_target,
```

Comment:
```suggestion
        llvm_target: super::apple_base::ios_llvm_target("arm64"),
```
Nit: no need to create a separate variable (here and in other files as well).